doc/Command-Reference.md

```diff
@@ -7961,8 +7962,292 @@ Go Back To [Beginning of the document](#) or [Beginning of this section](#waterm
 
 ## Software Installation and Management
 
-SONiC software can be installed in two methods, viz, "using sonic-installer tool", "ONIE Installer".
+SONiC software image can be installed in two methods, viz, "using sonic-installer tool", "ONIE Installer".
 
+SONiC feature Docker images (aka "SONiC packages") available to be installed with *sonic-package-manager* utility.
+
+### SONiC Package Manager
+
+This is a command line tool that provides functionality to manage SONiC Packages on SONiC device.
+
+**sonic-package-manager list**
+
+This command lists all available SONiC packages, their desription, installed version and installation status.
+SONiC package status can be *Installed*, *Not installed* or *Built-In*. "Built-In" status means that a feature is built-in to SONiC image and can't be upgraded or uninstalled.
+
+- Usage:
+  ```
+  sonic-package-manager list
```

Comment:
Guess "sonic"  is implied. It could be part of show command

"package list"

Reply:
This is on par with "sonic-installer".
It can be later added in show command, similar to "show boot", which calls "sonic-installer list".